docs/readmes/contributing/contribute_onboarding.md

```diff
@@ -6,26 +6,26 @@ hide_title: true
 
 # Developer Onboarding
 
-This document walks a new developer through the process of onboarding to Magma, specifically to the Orc8r subproject within the Magma project.
+This document walks a new developer through the process of onboarding to the Magma project.
 
 ## Project overview
 
 Magma is an open-source software platform that gives network operators an open, flexible, and extendable mobile core network solution. Our mission is to connect the world to a faster network by enabling service providers to build cost-effective and extensible carrier-grade networks.
 
 In more approachable terms, Magma is a collection of software that makes running things like a cellular network affordable and customizable. With Magma's current rate of growth, increasing stability and reliability of the platform are all huge wins.
 
-There are three main components in the Magma software platform: Access Gateway (AGW), Orc8r (Orc8r), and Federation Gateway (FeG). If you want to read more, see the [Magma intro docs](https://magma.github.io/magma/docs/next/basics/introduction.html).
+There are three main components in the Magma software platform: Access Gateway (AGW), Orchestrator (Orc8r), and Federation Gateway (FeG). If you want to read more, see the [Magma intro docs](https://magma.github.io/magma/docs/next/basics/introduction.html).
+
+### Access Gateway
+
+The AGW defines datapath rules for connecting subscribers through to the Internet. It pulls configuration from Orc8r, sets datapath rules, manages charging and accounting, reports state and metrics to Orc8r, and more.
```

Comment:
We should reword this, user might get confused with the wording. AGW provides the mobile core functionality. It contains the components which implements EPC (LTE) and 5GC. We can add more details here.

Reply:
Yeah the AGW+FeG sections are mostly placeholders, feel free to totally replace 🙂